scripts/create_xpi.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Assuming this is run from repo root
+zip -jr zip/cmunits.xpi src
```

Comment:
Please change to:

	test $$(git config user.name) || git config user.name "Build bot (via TravisCI)"
	test $$(git config user.email) || git config user.email "build-bot@travis"
        zip -jr docs/cmunit.xpi src
        git add docs/cmunit.xpi
        git commit -m "build xpi extension"
        git push https://${GH_TOKEN}@github.com/cmu-student-government/cmunit.git master

This is will be a lot easier to use from CI